backend/Dockerfile.persistenceagent

```diff
@@ -30,7 +30,11 @@ RUN go-licenses csv ./backend/src/agent/persistence > /tmp/licenses.csv && \
     diff /tmp/licenses.csv backend/third_party_licenses/persistence_agent.csv && \
     go-licenses save ./backend/src/agent/persistence --save_path /tmp/NOTICES
 
-FROM alpine:3.8
+FROM alpine:3
```

Comment:
Same as in [here](https://github.com/kubeflow/pipelines/pull/8944/files#r1131848819)

Reply:
same as in [here](https://github.com/kubeflow/pipelines/pull/8944#discussion_r1131862439)